src/MessageHandler/DatasetSubmissionFilerHandler.php

```diff
@@ -70,30 +77,43 @@ class DatasetSubmissionFilerHandler implements MessageHandlerInterface
      */
     private $datastore;
 
+    /**
+     * Datastore directory.
+     *
+     * @var string
+     */
+    private $datastoreDirectory;
+
     /**
      * DatasetSubmissionFilerHandler constructor.
      *
      * @param DatasetSubmissionRepository $datasetSubmissionRepository Dataset Submission Repository.
      * @param LoggerInterface             $filerLogger                 Name hinted filer logger.
+     * @param LoggerInterface             $virusScanLogger             Name hinted virus-scan logger.
```

Comment:
This is is submissions handler, virus scan logger should go to it's own handler.